bin-src/lib/getDependentStoryFiles.ts

```diff
@@ -213,6 +278,7 @@ export async function getDependentStoryFiles(
     checkedIds[id] = true;
     reasonsById[id].filter(untrace).forEach((reason) => traceName(reason, tracePath));
   }
+  // TODO: should rename (this is webpack modules) -- maybe affectedStoryFiles?
```

Comment:
You might want to consider calling it `changedPackages` instead of `changedModules` also. Even though the directory is called `node_modules`, I'm pretty sure when people say "module" they usually mean *a single file*. Eg, from [the node docs](https://nodejs.org/api/modules.html):

> In Node.js, each file is treated as a separate module

And ESM definitely is referring to single files as the ES "modules". Webpack too calls each file a module. I'm not sure what the history of "`node_modules`" is, but it was a mistake, I guess!